backend/buildeR.BLL/Services/GroupService.cs

```diff
@@ -41,7 +43,21 @@ public async Task<GroupDTO> Create(NewGroupDTO group)
             {
                 throw new ArgumentNullException();
             }
-            return await base.AddAsync(group);
+
+            var newGroup = await base.AddAsync(group);
+
+            var newTeamMember = new NewTeamMemberDTO
+            {
+                UserId = group.CreatorId,
+                MemberRole = UserRole.Creator,
+                GroupId = newGroup.Id
+            };
+
+            var teamMember = Mapper.Map<TeamMember>(newTeamMember);
```

Comment:
Why do map the object? Create TeamMember entity instead of dto. Add a field JoinedDate to save when the user was added to the group